src/common.js

```diff
@@ -183,11 +183,12 @@ function parseOptions(opt, map, errorOptions) {
   });
 
   if (!opt) return options; // defaults
+  var hasErrorOpts = typeof errorOptions === 'object';
```

Comment:
I would be careful here, because `typeof null === 'object'`. This check exists elsewhere in this function (and in `wrap`).